docs/websites/varnish/use-varnish-with-nginx-to-serve-cached-wordpress-content-over-ssl-on-debian-8.md

```diff
@@ -0,0 +1,683 @@
+
+Both Varnish and nginx are versatile tools with a variety of uses. This guide will present a basic setup that you can refine to meet your specific needs.
+
+## Our setup
+
+For this example, we will configure nginx and Varnish for two Wordpress sites:
+
+  * `www.example-over-http.com` will be an unencrypted, HTTP-only site.
+  * `www.example-over-https.com` will be a separate, HTTPS-encrypted site.
+
+For HTTP traffic Varnish will listen on port 80. If content is found in cache, Varnish will serve it from cache. If not, it will pass the request to nginx on port 8080. nginx will send the requested content back to Varnish on the same port, and Varnish will store the fetched content in cache and then deliver it to the client on port 80.
+
+For HTTPS traffic, nginx will listen on port 443 and send decrypted traffic to Varnish on port 80. If content is found in cache, Varnish will send the unencrypted content from cache back to nginx, which will encrypt it and send it to the client. If content is not found in cache Varnish will request it from backend nginx on port 8080, store it in cache and then send it unencrypted to frontend nginx which will encrypt it and send it to the client's browser.
+
+Our setup is illustrated below. Please note that frontend nginx and backend nginx are one and the same server:
+
+[![Nginx-Varnish-Nginx server configuration diagram](/docs/assets/varnish-scheme-small.png)](/docs/assets/varnish-scheme.png)
+
+## Before You Begin
+
+This tutorial assumes that you have SSH access to your Linode running Debian 8 (Jessie). Before you get started, make sure you've completed the following:
+
+*   Complete the steps in our [Getting Started](https://linode.com/docs/getting-started) and [Securing your Server](https://www.linode.com/docs/security/securing-your-server) guides.
+
+*   Follow the steps outlined in our [LEMP on Debian 8](https://www.linode.com/docs/websites/lemp/lemp-server-on-debian-8). You should skip the nginx configuration section as we'll address it later on in our tutorial.
+
+*   Follow the steps in our [WordPress](https://www.linode.com/docs/websites/cms/how-to-install-and-configure-wordpress) guide to install and configure WordPress.
+
+## Install and Configure Varnish
+
+1.  Issue the following commands to install Varnish:
+
+		sudo apt-get update
+		sudo apt-get install varnish
+
+2.  Use 'nano' or other text editor to edit the `/etc/default/varnish` file:
+
+		sudo nano /etc/default/varnish
+
+3.  To make Varnish start at boot, under `Should we start varnishd at boot?` set the `START` parameter to `yes`:
+
+		START=yes
+
+4. In the `Alternative 2` section, make the following changes to the `DAEMON_OPTS` section:
+
+	{: .file-excerpt }
+	/etc/default/varnish
+	:   ~~~ conf
+	
+		DAEMON_OPTS="-a :80 \
+				 	-T localhost:6082 \
+				 	-f /etc/varnish/custom.vcl \
+				 	-S /etc/varnish/secret \
+				 	-s malloc,500M"
```

Comment:
Two quick questions here. 

1) Now that the minimum Linode plan has 2 gigabytes of RAM, can/should we update this variable?
2) By default, the configuration file uses a lowercase `m`. Is there any difference as far as Varnish is concerned, or any other reason to change it here?